lib/google.js

```diff
@@ -22,18 +26,31 @@ module.exports.reset = function () {
 };
 
 module.exports.readConfig = function (configIn) {
-	config = configIn;
-	if (!config) {
+	if (!configIn) {
 		// no google iap or public key(s) from ENV variables
 		return;
 	}
+    config = {};
+    var config_value_set = false;
```

Comment:
I am being a little picky here :-)
Would  you change the variable name to configValueSet? I would like to use camel casing for all variable names.